### DISCUSSION TITLE
Approved status should be refurbished for edge situations in which Ranked status would be controversial.

### DISCUSSION THREAD

**Discussion by RaFaReAcH** [ppy/osu#24791] [Ideas]
The following is all my opinion:

There has been no shortage of controversial maps, usually top diffs, that tether the line of what can be ranked and push the boundries of what is acceptable as a ranked map, this sounds good on paper, but in practice it creates divide in the community and homogenices the ranked section into a mess of "anything goes", which is fine to some extent but as proved by the existence of the Loved category, some maps were simply too much/deserved recognition but simply were not Ranked status material.

However, Loved status was brought upon with a completely different process, completely different criteria, completely different everything, so it is good at catching really edge scenarios, but gray areas are completely unaccounted for, which leads for the funny drama on some maps.

With the recent discussion regarding the newly DQ'd HP0 map [Smile of the End](https://osu.ppy.sh/beatmapsets/2006974#osu/4174539), I thought it would be a good time to bring it up, my proposal that I have had for years and really wanted to argue about for a good while is, since it is literally a dead category that was used to handle edge scenarios before, why not bring back the Approved category to give to these maps that would have gotten a controversial Ranked otherwise? There is no functional difference aside from distancing them from the ranked section in which the "normal" and "good" maps are supposed to be found, pushing for any difference would be a bit meaningless, since any "cheese" you may get from any future map in this category could most likely be found in a previous already ranked map to exploit.

No mechanic difference and no different process to go through, simply set a map as Approved when deemed necessary, seems like a pointless change but really it would easily make everyone stop complaining about edge scenarios on the Ranked section and prevent even more disagreements over what the Ranked section is even supposed to be about anymore, and it is in line with the spirit of the Approved section which was basically "Yo we can't let this into Ranked or people would complain (inflated score back then) so let's just..."

I can't think of any reason NOT to do it, you don't even need to retroactively apply it to previous controversial maps necessarily, it can just keep "WHY DID THIS MAP GET RANKED AAAAAAAAAAAAAAAA" controvercy away.

Some examples off the top of my head that, had they gotten Approved under this definition, no one would have cared to cry about: [Oshama Scramble! (IOException Edit)](https://osu.ppy.sh/beatmapsets/1376308#osu/2849194) , [HAWATARI NIOKU CENTI (TV Size), AKA: The CON map](https://osu.ppy.sh/beatmapsets/1893971#osu/3977191) , [Xeroa PREON](https://osu.ppy.sh/beatmapsets/1487999#osu/3050529) , [Shiten](https://osu.ppy.sh/beatmapsets/1308171#osu/2772606) (really both sets), next two I don't think are bad in hindsight but that is just because my view has been changed by how different Ranked is now from back then: [Apparition](https://osu.ppy.sh/beatmapsets/216979#osu/690982) and [A-L-I-E-N](https://osu.ppy.sh/beatmapsets/456366#osu/978026).

**Comment by bdach** [comment]
For what it's worth, the last time this sort of thing came up the decision was that [approved is not to be used anymore](https://github.com/ppy/osu-web/issues/8436#issuecomment-995393223).

**Reply by RaFaReAcH** [reply]
Well, I am unaware of what plans there are for the Approved section, it has been long, so this is assuming it doesn't clash with future plans.

On another note I was told this is more of an osu forum discussion kind of topic, posting it there and following the steps detailed properly.

**Comment by Nivalyx** [comment]
The idea (and the intent behind it) is definitely something that warrants further discussion in regards to the current state of the Ranked section, however I think the entire suggestion falls short on one aspect — *"how should 'controversial' be defined?"*

It is important to consider that mapping is always evolving, and we've seen time and again how maps such as [Miss You](http://osu.ppy.sh/s/155691) and [Rising Hope](http://osu.ppy.sh/s/157871) went from "incredibly controversial" during its era to "just another map in the Ranked section" today.

Unless we (as a community) can all agree to a foolproof, objective definition of "controversial", I don't think this solution works as an answer of the problem.

**Reply by RaFaReAcH** [reply]
You bring up a good comment, as I said previously, the lazer repertoir does not seem to be quite the place for it, so I have created a [forum post](https://osu.ppy.sh/community/forums/topics/1820455?n=1) detailing basically the same I have here, may I direct you to comment your concerns there so we can get it all in one place?

**Reply by Nivalyx** [reply]
kk
